app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file_to_move, new_file = command.split()
+    if command_name == "mv":
+        whole_data = ""
+        with open(file_to_move) as file:
+            whole_data = file.read()
+
+        os.remove(file_to_move)
+        path_to_new_file = new_file.split("/")
+
+        combined_path = ""
+        for path in path_to_new_file[:-1]:
+            combined_path = os.path.join(combined_path, path)
```

Comment:
better to use `os.path.split`

Reply:
but I don't need to split my path here, I need to do the opposite, to join the final path